### ISSUE TITLE
[BUG] Getting exception with new net5 using winform together with rxui

### ISSUE THREAD

**Issue by ScarletKuro** [reactiveui/ReactiveUI#2577]
**Describe the bug**

RxUI doesn't support net5 with winform? I get exception above, but if i remove in my view Whenactivated and binding inside then form starts to work. Adding *System.Runtime.InteropServices.WindowsRuntime* to the project didn't help.
**Exception**
```
 ---> System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime.InteropServices.WindowsRuntime, Version=4.0.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
File name: 'System.Runtime.InteropServices.WindowsRuntime, Version=4.0.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at System.Reactive.ReflectionUtils.GetEventMethods[TSender,TEventArgs](Type targetType, Object target, String eventName, MethodInfo& addMethod, MethodInfo& removeMethod, Type& delegateType, Boolean& isWinRT)
   at System.Reactive.Linq.QueryLanguage.FromEventPattern_[TSender,TEventArgs,TResult](Type targetType, Object target, String eventName, Func`3 getResult, IScheduler scheduler) in /_/Rx.NET/Source/src/System.Reactive/Linq/QueryLanguage.Events.cs:line 263
   at System.Reactive.Linq.QueryLanguage.FromEventPattern_[TEventArgs](Object target, String eventName, IScheduler scheduler) in /_/Rx.NET/Source/src/System.Reactive/Linq/QueryLanguage.Events.cs:line 172
   at System.Reactive.Linq.QueryLanguage.FromEventPattern[TEventArgs](Object target, String eventName) in /_/Rx.NET/Source/src/System.Reactive/Linq/QueryLanguage.Events.cs:line 160
   at System.Reactive.Linq.Observable.FromEventPattern[TEventArgs](Object target, String eventName) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable.Events.cs:line 648
   at ReactiveUI.Winforms.CreatesWinformsCommandBinding.BindCommandToObject[TEventArgs](ICommand command, Object target, IObservable`1 commandParameter, String eventName) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI.Winforms\CreatesWinformsCommandBinding.cs:line 100
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at ReactiveUI.Winforms.CreatesWinformsCommandBinding.BindCommandToObject(ICommand command, Object target, IObservable`1 commandParameter) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI.Winforms\CreatesWinformsCommandBinding.cs:line 74
   at ReactiveUI.CreatesCommandBinding.BindCommandToObject(ICommand command, Object target, IObservable`1 commandParameter) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Bindings\Command\CreatesCommandBinding.cs:line 51
   at ReactiveUI.CommandBinderImplementation.<>c__DisplayClass2_0`3.<BindCommandInternal>b__3(<>f__AnonymousType0`2 x) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Bindings\Command\CommandBinderImplementation.cs:line 180
   at System.Reactive.AnonymousSafeObserver`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/AnonymousSafeObserver.cs:line 43
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Where`1.Predicate._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Where.cs:line 54
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.CombineLatest`3._.SecondObserver.OnNext(TSecond value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/CombineLatest.cs:line 177
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 47
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.DistinctUntilChanged`2._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/DistinctUntilChanged.cs:line 72
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 47
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Where`1.Predicate._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Where.cs:line 54
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Switch`1._.InnerObserver.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Switch.cs:line 100
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 47
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.IdentitySink`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/Internal/IdentitySink.cs:line 15
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.ToObservableRecursive`1._.LoopRec(IScheduler scheduler) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/ToObservable.cs:line 114
   at System.Reactive.Linq.ObservableImpl.ToObservableRecursive`1._.<>c.<Run>b__3_0(IScheduler innerScheduler, _ this) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/ToObservable.cs:line 55
   at System.Reactive.Concurrency.ImmediateScheduler.Schedule[TState](TState state, Func`3 action) in /_/Rx.NET/Source/src/System.Reactive/Concurrency/ImmediateScheduler.cs:line 41
   at System.Reactive.Linq.ObservableImpl.ToObservableRecursive`1._.Run(IEnumerable`1 source, IScheduler scheduler) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/ToObservable.cs:line 55
   at System.Reactive.Linq.ObservableImpl.ToObservableRecursive`1.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/ToObservable.cs:line 24
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.TailRecursiveSink`1.Drain() in /_/Rx.NET/Source/src/System.Reactive/Internal/TailRecursiveSink.cs:line 126
   at System.Reactive.TailRecursiveSink`1.Run(IEnumerable`1 sources) in /_/Rx.NET/Source/src/System.Reactive/Internal/TailRecursiveSink.cs:line 32
   at System.Reactive.Linq.ObservableImpl.Concat`1.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Concat.cs:line 20
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 22
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Linq.ObservableImpl.Switch`1._.OnNext(IObservable`1 value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Switch.cs:line 58
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 47
   at System.Reactive.Linq.ObservableImpl.ReturnImmediate`1.Run(IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Return.cs:line 62
   at System.Reactive.BasicProducer`1.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 66
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 22
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.Switch`1.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Switch.cs:line 20
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.Where`1.Predicate.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Where.cs:line 27
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 22
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.DistinctUntilChanged`2.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/DistinctUntilChanged.cs:line 24
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.Select`2.Selector.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Select.cs:line 22
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Linq.ObservableImpl.CombineLatest`3._.Run(IObservable`1 first, IObservable`1 second) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/CombineLatest.cs:line 57
   at System.Reactive.Linq.ObservableImpl.CombineLatest`3.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/CombineLatest.cs:line 29
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 125
   at System.ObservableExtensions.SubscribeSafe[T](IObservable`1 source, IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 388
   at System.Reactive.Sink`2.Run(IObservable`1 source) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 89
   at System.Reactive.Linq.ObservableImpl.Where`1.Predicate.Run(_ sink) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Where.cs:line 27
   at System.Reactive.Producer`2.<>c.<SubscribeRaw>b__1_0(ValueTuple`2 tuple) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 121
   at System.Reactive.Concurrency.Scheduler.<>c__75`1.<ScheduleAction>b__75_0(IScheduler _, ValueTuple`2 tuple) in /_/Rx.NET/Source/src/System.Reactive/Concurrency/Scheduler.Simple.cs:line 65
   at System.Reactive.Concurrency.CurrentThreadScheduler.Schedule[TState](TState state, TimeSpan dueTime, Func`3 action) in /_/Rx.NET/Source/src/System.Reactive/Concurrency/CurrentThreadScheduler.cs:line 101
   at System.Reactive.Concurrency.LocalScheduler.Schedule[TState](TState state, Func`3 action) in /_/Rx.NET/Source/src/System.Reactive/Concurrency/LocalScheduler.cs:line 32
   at System.Reactive.Concurrency.Scheduler.ScheduleAction[TState](IScheduler scheduler, TState state, Action`1 action) in /_/Rx.NET/Source/src/System.Reactive/Concurrency/Scheduler.Simple.cs:line 61
   at System.Reactive.Producer`2.SubscribeRaw(IObserver`1 observer, Boolean enableSafeguard) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 119
   at System.Reactive.Producer`2.Subscribe(IObserver`1 observer) in /_/Rx.NET/Source/src/System.Reactive/Internal/Producer.cs:line 97
   at System.ObservableExtensions.Subscribe[T](IObservable`1 source, Action`1 onNext) in /_/Rx.NET/Source/src/System.Reactive/Observable.Extensions.cs:line 63
   at ReactiveUI.CommandBinderImplementation.BindCommandInternal[TView,TProp,TParam](IObservable`1 this, TView view, Expression controlExpression, IObservable`1 withParameter, String toEvent, Func`2 commandFixuper) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Bindings\Command\CommandBinderImplementation.cs:line 162
   at ReactiveUI.CommandBinderImplementation.BindCommand[TView,TViewModel,TProp,TControl,TParam](TViewModel viewModel, TView view, Expression`1 vmProperty, Expression`1 controlProperty, IObservable`1 withParameter, String toEvent) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Bindings\Command\CommandBinderImplementation.cs:line 134
   at ReactiveUI.CommandBinderImplementationMixins.BindCommand[TView,TViewModel,TProp,TControl](ICommandBinderImplementation this, TViewModel viewModel, TView view, Expression`1 propertyName, Expression`1 controlName, String toEvent) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Bindings\Command\CommandBinderImplementationMixins.cs:line 33
   at ReactiveUI.CommandBinder.BindCommand[TView,TViewModel,TProp,TControl](TView view, TViewModel viewModel, Expression`1 propertyName, Expression`1 controlName, String toEvent) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Bindings\Command\CommandBinder.cs:line 95
   at WinPOS.Nugis.Client.Forms.Window.CustomerInputWindow.<.ctor>b__0_0(CompositeDisposable disposable) in C:\Users\KuroPC\source\repos\Kiisu5\src\WinPOS.Kiisu.UI.Winform\Forms\Window\CustomerInputWindow.cs:line 21
   at ReactiveUI.ViewForMixins.<>c__DisplayClass9_0.<WhenActivated>b__0() in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Activation\ViewForMixins.cs:line 238
   at ReactiveUI.ViewForMixins.<>c__DisplayClass10_0.<HandleViewActivation>b__0(Boolean activated) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Activation\ViewForMixins.cs:line 255
   at System.Reactive.AnonymousSafeObserver`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/AnonymousSafeObserver.cs:line 43
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.DistinctUntilChanged`2._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/DistinctUntilChanged.cs:line 72
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Merge`1.Observables._.InnerObserver.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Merge.cs:line 236
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.DistinctUntilChanged`2._.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/DistinctUntilChanged.cs:line 72
   at System.Reactive.Sink`1.ForwardOnNext(TTarget value) in /_/Rx.NET/Source/src/System.Reactive/Internal/Sink.cs:line 49
   at System.Reactive.Linq.ObservableImpl.Merge`1.Observables._.InnerObserver.OnNext(TSource value) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable/Merge.cs:line 236
   at System.Reactive.Subjects.Subject`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/Subjects/Subject.cs:line 146
   at ReactiveUI.Winforms.ActivationForViewFetcher.<>c__DisplayClass2_3.<GetActivationForView>b__10(Object sender, EventArgs e) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI.Winforms\ActivationForViewFetcher.cs:line 50
   at System.Windows.Forms.Control.OnHandleCreated(EventArgs e)
   at System.Windows.Forms.Form.OnHandleCreated(EventArgs e)
   at System.Windows.Forms.Control.WmCreate(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ContainerControl.WndProc(Message& m)
   at System.Windows.Forms.Form.WmCreate(Message& m)
   at System.Windows.Forms.Form.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime.InteropServices.WindowsRuntime, Version=4.0.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.
File name: 'System.Runtime.InteropServices.WindowsRuntime, Version=4.0.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
   at System.Reactive.ReflectionUtils.GetEventMethods[TSender,TEventArgs](Type targetType, Object target, String eventName, MethodInfo& addMethod, MethodInfo& removeMethod, Type& delegateType, Boolean& isWinRT)
   at System.Reactive.Linq.QueryLanguage.FromEventPattern_[TSender,TEventArgs,TResult](Type targetType, Object target, String eventName, Func`3 getResult, IScheduler scheduler) in /_/Rx.NET/Source/src/System.Reactive/Linq/QueryLanguage.Events.cs:line 263
   at System.Reactive.Linq.QueryLanguage.FromEventPattern_[TEventArgs](Object target, String eventName, IScheduler scheduler) in /_/Rx.NET/Source/src/System.Reactive/Linq/QueryLanguage.Events.cs:line 172
   at System.Reactive.Linq.QueryLanguage.FromEventPattern[TEventArgs](Object target, String eventName) in /_/Rx.NET/Source/src/System.Reactive/Linq/QueryLanguage.Events.cs:line 160
   at System.Reactive.Linq.Observable.FromEventPattern[TEventArgs](Object target, String eventName) in /_/Rx.NET/Source/src/System.Reactive/Linq/Observable.Events.cs:line 648
```

**Steps To Reproduce**
1. Create new winform project using net 5
2. Add ReactiveUI, ReactiveUI.WinForm, ReactiveUI.Events,Winforms, ReactiveUI.Foddy
3. Make a standart view with WhenActivated and bindcommand
4. Open form with ShowDialog()

**Expected behavior**
To work with new net5.

**Environment**

- OS:  Windows 10
- Version: ReactiveUI 12.1.5, ReactiveUI.WinForm 12.1.5, ReactiveUI.Events,Winforms 12.1.5, ReactiveUI.Foddy

**Comment by glennawatson** [comment]
I plan on adding .net 5 support today most likely and do another release.

Thanks for the report.

**Comment by fugaku** [comment]
Got the same error on WPF for net5

**Comment by glennawatson** [comment]
Yep, .net 5 support hasn't been turned on yet slowly porting all the projects over.

**Comment by C1rdec** [comment]
Any updates?

**Comment by glennawatson** [comment]
Got some unit tests to fix on dynamic data and then rxui conversion can start. It's not trivial.

**Comment by MovGP0** [comment]
Apparently the `DispatcherScheduler` for WPF class has been removed in .NET 5.0. Getting this error:
```
System.TypeInitializationException
  HResult=0x80131534
  Message=The type initializer for 'ReactiveUI.RxApp' threw an exception.
  Source=ReactiveUI
  StackTrace:
   at ReactiveUI.RxApp.set_TaskpoolScheduler(IScheduler value) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\RxApp.cs:line 189
   at ReactiveUI.PlatformRegistrations.Register(Action`2 registerFunction) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Platforms\netcoreapp3\PlatformRegistrations.cs:line 30
   at ReactiveUI.DependencyResolverMixins.InitializeReactiveUI(IMutableDependencyResolver resolver, RegistrationNamespace[] registrationNamespaces) in D:\a\ReactiveUI\ReactiveUI\src\ReactiveUI\Mixins\DependencyResolverMixins.cs:line 57

Inner Exception 1:
TypeLoadException: Could not load type 'System.Reactive.Concurrency.DispatcherScheduler' from assembly 'System.Reactive, Version=5.0.0.0, Culture=neutral, PublicKeyToken=94bc3704cddfc263'.
```

**Comment by C1rdec** [comment]
If I remember correctly `ObserveOnDispatcher()` is an extension method defined in `System.Reactive.Windows.Threading` that can be used.

**Comment by glennawatson** [comment]
DynamicData was updated yesterday so started work on rxui now .

**Comment by glennawatson** [comment]
You can get around this most likely by doing `net5.0-windows10.0.19041` as your TargetFramework.

I've got a workaround for that in the upcoming PR but this might solve your problem in the short term.

**Comment by glennawatson** [comment]
Fixed by #2589
